docs/usage.rst

```diff
@@ -94,6 +95,24 @@ As of version 3.0.0, ``mocker.spy`` also works with ``async def`` functions.
 
     .. _#175: https://github.com/pytest-dev/pytest-mock/issues/175
 
+As of version 3.9.1, spying can be also selectively stopped.
```

Comment:
Given this is a new feature, the next will be a minor release.

```suggestion
As of version 3.10, spying can be also selectively stopped.
```

Reply:
Indeed, 3.10 or rather 3.10.0 ?